Swiften/Parser/PayloadParsers/MIXUserPreferenceParser.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2017 Tarun Gupta
+ * Licensed under the simplified BSD license.
+ * See Documentation/Licenses/BSD-simplified.txt for more information.
+ */
+
+#include <Swiften/Parser/PayloadParsers/MIXUserPreferenceParser.h>
+
+#include <boost/optional.hpp>
+
+#include <Swiften/Parser/PayloadParserFactory.h>
+#include <Swiften/Parser/PayloadParsers/FormParser.h>
+
+using namespace Swift;
```

Comment:
Please uses `namespace Swift {` instead of `using namespace Swift` in implementation files. Happy for you to use `using namespace Swift` in unit tests though.